tests/opentelemetry-test-utils/setup.cfg

```diff
@@ -40,9 +40,10 @@ packages=find_namespace:
 install_requires =
     opentelemetry-api ~= 1.3
     opentelemetry-sdk ~= 1.3
+    asgiref ~= 3.0
 
 [options.extras_require]
-test = flask~=2.0
+test =
```

Comment:
I'm confused by these changes, can you add a note in the description?

Reply:
Updated.